### PR TITLE
[GH-591] Make max txs per block be equal to epoch node

### LINKED ISSUE #591: Adjust the limit for transactions in a block
edit: as @gspasov said below, the limit should be changed from 100 to 10946

### PR DESCRIPTION
**Pull request by gspasov**
resolves #591

### CODE REVIEW

File: apps/aecore/lib/aecore/governance/governance_constants.ex [hunk 1]
```diff
@@ -29,6 +29,8 @@ defmodule Aecore.Governance.GovernanceConstants do
 
   @name_claim_burned_fee 3
 
+  @max_txs_per_block 10946
```
Comment (cytadela8) [inline apps/aecore/lib/aecore/governance/governance_constants.ex]: This is some duplication. Max txs per block should be defined in GovernaneConstants OR in config. Not both.

Comment (gspasov) [reply apps/aecore/lib/aecore/governance/governance_constants.ex]: Forgot to delete it.

**Comment by 0legIv** [comment]
@thepiwo I removed them just for now, there are some errors in the Multi node test framework now, we are investigating them with @gspasov

**Comment by thepiwo** [comment]
thank you @0legIv

File: apps/aetestframework/test/multi_node_sync_test.exs [hunk 2]
```diff
@@ -39,9 +36,7 @@ defmodule MultiNodeSyncTest do
   test "spend_tx test" do
     TestFramework.mine_sync_block("node1")
     TestFramework.spend_tx("node1")
-    assert TestFramework.get_pool_tx_count("node1") == 1
```
Comment (cytadela8) [inline apps/aetestframework/test/multi_node_sync_test.exs]: Why do we remove all those asserts?

Comment (cytadela8) [reply apps/aetestframework/test/multi_node_sync_test.exs]: How are we sure now that the txs get mined?

**Comment by thepiwo** [comment]
@gspasov this is ready, but just waiting for https://github.com/aeternity/elixir-node/pull/617 right?

**Comment by gspasov** [comment]
Yes, since it has some already unwanted changes to the TestFramework

**Comment by thepiwo** [comment]
thank you @gspasov

**Pull request merged**